src/player.c

```diff
@@ -1678,23 +1678,21 @@ get_status(void *arg, int *retval)
 
   status->plid = cur_plid;
 
-  if (pb_session.playing_now == NULL)
-    memset(&status->quality, 0, sizeof(struct media_quality));
-  else
-    {
-      status->quality.sample_rate     = pb_session.playing_now->quality.sample_rate;
-      status->quality.bits_per_sample = pb_session.playing_now->quality.bits_per_sample;
-      status->quality.channels        = pb_session.playing_now->quality.channels;
-      status->quality.bit_rate        = pb_session.playing_now->quality.bit_rate;
-      status->quality.codec           = pb_session.playing_now->quality.codec;
-    }
-
   switch (player_state)
     {
       case PLAY_STOPPED:
 	DPRINTF(E_DBG, L_PLAYER, "Player status: stopped\n");
 
 	status->status  = PLAY_STOPPED;
+	if (pb_session.playing_now)
```

Comment:
Remove this. If player_state is PLAY_STOPPED then playing_now should always be null (unless there is a bug).

Reply:
With the check removed the server SEGVs on startup with the web interface connecting.

Reply:
I meant remove the entire if clause, i.e. leave case PLAY_STOPPED as it was before